src/Card/index.js

```diff
@@ -165,13 +168,16 @@ class Card extends React.Component<Props, State> {
 
   hasAdjustedHeader = () => {
     const children = this.getChildren();
+    if (children === undefined) {
```

Comment:
I believe `[]` is not returned when  using optional chaining as it's happening in Line 137 and 140 above. IE is returning some pretty unpredictable object data :confounded:

Reply:
I can add an ` || [] ` to those returns if you prefer

Reply:
![image](https://user-images.githubusercontent.com/7850592/50596421-8e84bd00-0ea4-11e9-9dd9-c318ee652f1a.png) oohh you are right I didn't catch the index after children, so could you solve this on line `124` by extending condition please ? 🙏

Reply:
By `extending condition` you mean what I suggested above? ie. `return option.al?.chainig? || [] ` instead of returning the optional chain directly?
Or something else? :slightly_smiling_face:

Reply:
I mean something like

```
if(children === undefined || children[0] === undefined) ....
```

on line 124

Reply:
I don't think `React.Children.toArray` returns `undefined` ever, does it?

Reply:
Nope you are right `React.Children.toArray(undefined)` returns `[]`, so we can let this condition here